src/dropdown.ts

```diff
@@ -2304,35 +2301,17 @@ export default class Dropdown extends LitElement implements FormControl {
   }
 
   #onOptionsSelectedChange(event: Event) {
-    if (!this.multiple) {
-      for (const option of this.#optionElements) {
```

Comment:
Moved this inside the conditional below that contains everything else to do with single-select.